From sshtylyov@ru.mvista.com Sun Apr  1 10:13:22 2007
From: Sergei Shtylyov <sshtylyov@ru.mvista.com>
Date: Sun, 1 Apr 2007 21:13:58 +0400
Subject: PCI: define pci_request/release_regions() for CONFIG_PCI=n
To: greg@kroah.com
Cc: linux-pci@atrey.karlin.mff.cuni.cz, bunk@stusta.de, akpm@osdl.org
Message-ID: <200704012113.58173.sshtylyov@ru.mvista.com>
Content-Disposition: inline


Balance declarations of pci_request_regions() and pci_release_regions() with
empty inline definitions for the CONFIG_PCI=n case -- otherwise my patch to
drivers/net/3c59x.c in the -mm tree doesn't compile. :-)

Signed-off-by: Sergei Shtylyov <sshtylyov@ru.mvista.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/pci.h |    3 +++
 1 file changed, 3 insertions(+)

--- a/include/linux/pci.h
+++ b/include/linux/pci.h
@@ -744,6 +744,9 @@ static inline int pci_set_power_state(st
 static inline pci_power_t pci_choose_state(struct pci_dev *dev, pm_message_t state) { return PCI_D0; }
 static inline int pci_enable_wake(struct pci_dev *dev, pci_power_t state, int enable) { return 0; }
 
+static inline int pci_request_regions(struct pci_dev *dev, const char *res_name) { return -EIO; }
+static inline void pci_release_regions(struct pci_dev *dev) { }
+
 #define pci_dma_burst_advice(pdev, strat, strategy_parameter) do { } while (0)
 
 static inline void pci_block_user_cfg_access(struct pci_dev *dev) { }
